City-of-Pori/pori-kada
# PORI-245 top menu to mobile menu

To test:

1. drush cc all
2. Make sure that top menu items are included in the mobile menu in pori.fi and visitpori.fi
3. Make sure the styles on the mobile menu work nicely (added some slight improvements)
